source/developersguide/dev.rst

```diff
@@ -255,7 +255,7 @@ encoding:
 
 .. parsed-literal::
       
-   >>> sig_str='&'.join(['='.join([k.lower(),urllib.quote_plus(request[k].lower().replace('+','%20'))])for k in sorted(request.iterkeys())]) 
+   >>> sig_str='&'.join(['='.join([k.lower(),urllib.quote_plus(request[k]).lower().replace('+','%20')])for k in sorted(request.iterkeys())]) 
```

Comment:
For reference, this is how cloudmonkey (python version) does it: https://github.com/apache/cloudstack-cloudmonkey/blob/5.3/cloudmonkey/requester.py#L203